image_op/reconstruct_traces.m

```diff
@@ -144,13 +148,21 @@ function reconstruct_traces(movie_source, ica_dir, varargin)
 info.num_pairs = rec_filter_count; %#ok<STRNU>
 filters = filters(:,:,1:rec_filter_count);
 
+cells_to_exclude = [];
 for cell_idx = 1:rec_filter_count % Normalize
     filter = filters(:,:,cell_idx);
-    filters(:,:,cell_idx) = filter / sum(filter(:));
+    sum_filter = sum(filter(:));
+    if sum_filter>0 && sum(filter(:)>0)>=min_num_pixels
+        filters(:,:,cell_idx) = filter / sum_filter;
+    else
+        cells_to_exclude(end+1) = cell_idx; 
+    end
 end
+filters(:,:,cells_to_exclude) = []; 
```

Comment:
If we're removing some filters this way, we then need to readjust `rec_filter_count` as:
```
rec_filter_count = rec_filter_count - length(cells_to_exclude);
info.num_pairs = rec_filter_count;
```